controller/controller_server/common_config.py

```diff
@@ -0,0 +1,11 @@
+import os.path
+
+import yaml
+from munch import DefaultMunch
+
+my_path = os.path.abspath(os.path.dirname(__file__))

```

Comment:
@oriyarde @matancarmeli7 

1. should I make those vars private?
2. should I move the `common_config` file to `/controller/common` and name it `config`?

Reply:
sounds good to me for both

Reply:
like @oriyarde

Reply:
done

Reply:
@oriyarde i know i opened it, plz resolve it